packages/javascript/src/__tests__/index.test.ts

```diff
@@ -167,4 +167,26 @@ describe("Appsignal", () => {
       expect(secondSpan.serialize().tags).toStrictEqual(testTag)
     })
   })
+
+  describe("addBreadcrumbs", () => {
```

Comment:
Can we test somewhere that the tags are sanitised? (The breadcrumb tags/metadata)

Reply:
Check! Fixed in a00b2b8.